extensions/azurecore/src/account-provider/auths/httpClient.ts

```diff
@@ -284,7 +285,7 @@ const networkRequestViaHttps = <T>(
 ): Promise<NetworkResponse<T>> => {
 	const isPostRequest = httpMethod === HttpMethod.POST;
 	const isPutRequest = httpMethod === HttpMethod.PUT;
-	const body: string = options?.body || '';
+	const body = (new TextEncoder()).encode(JSON.stringify(options?.body || ''));
```

Comment:
Why do we have to encode the body content? It looks like it accepts a utf8 string by default so is there something else that makes it so we need to encode it first?

https://nodejs.org/api/http.html#requestwritechunk-encoding-callback

Reply:
without encoding the body content, this table name caused an error to be thrown:

![MicrosoftTeams-image](https://user-images.githubusercontent.com/18150417/229945355-d345d92a-20ad-4c2a-a237-d9633e81dda8.png)

 and encoding fixed that error

Reply:
It's odd because I would think UTF-8 would be able to handle characters like that, so might be worth looking more into. 

But for now could you at least put a comment on all the places you use TextEncoder about why you're doing that so future people looking at this have context on why this is needed?